contrib/run_task.sh

```diff
@@ -1,13 +1,50 @@
-#!/bin/env bash
+#!/usr/bin/env bash
+#
+# We are attempting to run max 20 parallel jobs using GitHub actions (usage limit for free tier).
+#
+# ref: https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration
+#
+# The minimal/recent manifests are handled by CI (rust.yaml).
+#
+# Jobs get run twice, once for each manifest:
+#
+# 1+2  stable
+# 3+4  nightly
+# 5+6  msrv
+#
+# Jobs get run once:
+#
+# 7   lint
+# 8   docs
+# 9   docsrs
+# 10  bench
+# 11  ASAN
+# 12  WASM
+# 13  Schemars
+#
+# Jobs run by CI (rust.yml but without this script):
+#
+# 14  Kani
+# 15  Cross
+# 16  Arch32bit
+# 17  Embedded
+#
+# Jobs from other workflows:
+#
+# 18  coveralls.yml
+# 19  release.yml
+# 20  manage-pr.yml
 
 set -euox pipefail
 
+REPO_DIR=$(git rev-parse --show-toplevel)
+CRATES=("bitcoin" "hashes" "internals" "io" "units" "base58")
+
 # Make all cargo invocations verbose.
 export CARGO_TERM_VERBOSE=true
```

Comment:
Is this useful? I personally find the output way too noisy and have never used it to debug anything.

Reply:
I use it locally all the time because you can copy/paste its rustc invocations and then remove noise until you zero in on the problem.

It's sometimes useful in CI as well, though less so, as a way of diffing cargo invocations between versions.

Reply:
Cool, lets leave it in then.